rules/compose-pipe-style.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+const R = require('ramda');
+const isCalling = require('../ast-helper').isCalling;
+
+const startLine = R.path(['loc', 'start', 'line']);
+const endLine = R.path(['loc', 'end', 'line']);
+const isSingleLine = R.converge(R.equals, [startLine, endLine]);
+
+const create = context => ({
+    CallExpression(node) {
+        const matchCompose = isCalling({
+            name: 'compose'
+        });
+
+        const matchPipe = isCalling({
+            name: 'pipe'
+        });
+
+        if (matchCompose(node)) {
+            if (!isSingleLine(node)) {
```

Comment:
This could be simplified to `if (matchCompose(node) && !isSingleLine(node)) {`

Reply:
Of course! Overlooked it...